packages/cli/src/lib/deploy/PolywrapDeploy.ts

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+import { Deployer } from "./deployer";
+import { loadDeployManifest, loadDeployManifestExt } from "../project";
+import { CacheDirectory } from "../CacheDirectory";
+import { Logger } from "../logging";
+
+import { DeployManifest } from "@polywrap/polywrap-manifest-types-js";
+import { Schema as JsonSchema } from "jsonschema";
+import path from "path";
+import nodePath from "path";
+
+interface PolywrapDeployConfig {
+  cache: CacheDirectory;
+  logger: Logger;
+  defaultModulesCached: boolean;
+}
+
+export class PolywrapDeploy {
```

Comment:
We usually have `lib` classes like `Compiler`, `SchemaComposer`, etc. that the commands call.

Would it make sense to name this `ProjectDeployer` or simply `Deployer`?

I know `Deployer` already exists, although it does seem like that one should be called `DeployModule` rather than `Deployer`.

A small nitpick, just for clarity's sake 😄

Reply:
will resolve and conversation will follow in the other comment related to this